src/plugins/auto/src/plugin.cpp

```diff
@@ -360,6 +360,9 @@ IExecutableNetworkInternal::Ptr MultiDeviceInferencePlugin::LoadNetworkImpl(cons
     // updateFromMap will check config valid
     loadConfig.set_user_property(PreProcessConfig(config));
     loadConfig.apply_user_properties();
+    if (!setLogLevel(loadConfig.get_property(ov::log::level.name()).as<std::string>())) {
+        IE_THROW() << "Unsupported log level";
+    }
```

Comment:
then log from other calls except than loadmodel will still not getting outputs.

Reply:
Added judgment on whether log_level comes from user settings.